### ISSUE TITLE
test for over-fitting?

### ISSUE THREAD

**Issue by shadiakiki1986** [shadiakiki1986/keras-models-factory#5]
- perhaps I can add to `assert_fit_model` a few lines that check that the `val_loss` isn't much larger than the `loss`
  - the case of which would signify over-fitting
